locales/en/index.json

```
@@ -4648,6 +4648,19 @@
             "primaryAction": "Inizia",
             "secondaryAction": "Annulla"
           },
+          "NotTrustedRPScreen": {
```

Use camelCase for consistency
```suggestion
          "utrustedRpScreen": {
```